packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set dropdownDefault.twig__1.html

```diff
@@ -0,0 +1,6 @@
+<html><body>
+<div class="Dropdown">Content</div>
+<div id="DropdownExample" class="Dropdown">Content</div>
```

Comment:
Not a breaking issue but there are multiple duplicated ids within single HTML document. (Invalid HTML)

Reply:
Good point. Copy&paste bug :-) Fixed.